content/guides/tasks/index.md

```diff
@@ -0,0 +1,15 @@
+---
+rank: 195
+related_endpoints: []
+related_guides: []
+required_guides: []
+related_resources:
+  - task
+  - task_assignment
+alias_paths:
+---
+
+# Tasks
+
+Users can create tasks on files and assign them to collaborators on Box. Tasks
+can be used by developers to create file-centric workflows.
```

Comment:
Looking at my other feedback, I think we might want to take some time here to expand on the different task types (I think we have more than 1, right?) and what they look like in the UI. Maybe add a quick screenshot of one?

Reply:
Right now in the docs, we do not. We will have more than one in the future I think with Relay but haven't heard anything since the API change that was made to accommodate the coming relay changes

Reply:
Ok. I think regardless it might be good to expand on what the different resolution_states mean within the context of a review task?

Reply:
I'm expanding on this in the update-state guide. Do you want this on the index page?

Reply:
It might be nice to provide an overview of what a task is, maybe an image of it in action in Box, and then an overview of the task types, and there different states? This will be the first page people will visit when they want to learn about tasks.